velox/functions/sparksql/window/tests/NthValueTest.cpp

```diff
@@ -64,34 +64,23 @@ TEST_F(SparkNthValueTest, singlePartition) {
       {vectors}, "nth_value(c0, 1)", {overClause}, {frameClausee});
 }
```

Comment:
Please add a test in this file where int32 column is used for an offset parameter. Spark should not end up using it. But we should have a tested code-path.

Reply:
These changes were reverted recently and the tests are now moved SparkWindowTest.cpp.

Reply:
The intent was to ensure that velox function with int32 column offset works. This will not be used by Spark eval, but we should have a Velox test for it. Right now there aren't tests that use int32 column imo.

Velox unit test shouldn't be throwing any error. Spark will do its own error handling. And we don't use a Spark env in test framework.

Reply:
Are there any tests in SparkWindowTest.cpp for exercising the codepaths for int32 column offsets ?

Reply:
Thanks for the clarification. There were no tests to check for int32 column offset in SparkWindowTest.cpp and it is added now.